app/models/transcription_convention.rb

```diff
@@ -0,0 +1,25 @@
+class TranscriptionConvention < ApplicationRecord
+  belongs_to :institution
+
+  def self.create_default(institution_id)
+    default_list = [
+      ['Language', 'English (Australian)', '“Colour” not “Color”, “Organise” not “Organize”'],
+      ['Contractions', 'Transcribe as you hear them', '“Shoulda”, “Didn’t”'],
+      ['Numbers', 'Transcribe as numerals', '“11 West 40th Street”'],
+      ['Filled Pauses & Hesitations', 'Transcribe as you hear them', '“ah”, “eh”, “um”'],
+      ['Noise', 'Transcribe in brackets; use descriptive language', '“And then we [door slam]”, “So cold! [Brrrrr]”'],
+      ['Partial words', 'If someone stops speaking in the middle of a word, transcribe as much of the word as they say; follow it with a dash', '“Tes- Testing”, “Absolu- Absolutely”'],
+      ['Hard-to-understand', 'For speech that is difficult or impossible to understand, use question marks before and after', '“And she told me that ?I should just leave?”, “Her name was ?inaudible?”'],
+      ['Truncated Words', 'Sometimes words may seem to be repeated from the end of one line to the start of the next. Transcribe these as you hear them and use your best judgment', ''],
+    ]
```

Comment:
How about moving it to a `bootstrap` rake task or `db/seeds`?

Reply:
@lucascaton , this works as default values for an institution. So, I think having this as a seperate class gives more flexibility.

Reply:
Could it at least be a constant? :)